commands/commandimpls/account.go

```diff
@@ -193,7 +193,7 @@ func (account *Account) listAccounts(writer io.Writer) {
 	fmt.Fprintln(writer, "Available accounts:")
 	for _, acc := range config.Current.Accounts {
 		if acc.Token == config.Current.Token {
-			fmt.Fprintln(writer, "  [orange]> "+acc.Name+"[white]")
+			fmt.Fprintln(writer, "  ["+tviewutil.ColorToHex(config.GetTheme().AttentionColor)+"]> "+acc.Name+"[white]")
```

Comment:
Sorry ... should've mentioned it, but the same goes for "white".

Reply:
I mean... If I had more than 5 brain cells I would've thought of that myself, yea